test/Table_Tests/src/Common_Table_Operations/Conversion_Spec.enso

```diff
@@ -547,6 +549,15 @@ spec setup =
             t.auto_value_types . should_fail_with Unsupported_Database_Operation
             t.at "X" . auto_value_type . should_fail_with Unsupported_Database_Operation
 
+    if setup.is_database then Test.group prefix+"Table.from_objects" <|
+        Test.specify "should report unsupported" <|
+            Table.from_objects 12 . should_fail_with Unsupported_Database_Operation
+
+    if setup.is_database then Test.group prefix+"Table.expand_column" <|
+        Test.specify "should report unsupported" <|
+            table = table_builder [["aaa", [1, 2]], ["bbb", [3, 4]], ["ccc", [5, 6]]]
+            table.expand_column "bbb" . should_fail_with Unsupported_Database_Operation
```

Comment:
This test is good, but it would be worth to have a note here where are the related in-memory tests.

```suggestion
    # The in-memory functionality of `expand_column` is tested in  
test/Table_Tests/src/In_Memory/Table_Conversion_Spec.enso
    if setup.is_database then Test.group prefix+"Table.expand_column" <|
        Test.specify "should report unsupported" <|
            table = table_builder [["aaa", [1, 2]], ["bbb", [3, 4]], ["ccc", [5, 6]]]
            table.expand_column "bbb" . should_fail_with Unsupported_Database_Operation
```